DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -345,5 +367,28 @@ public IEnumerable<Course> GetCentrallyManagedAndCentreCourses(int centreId, int
                 new { centreId, categoryId }
             );
         }
+
+        public bool UpdateCourseOptions(CourseDetails courseDetails)

```

Comment:
On that note, I'm concerned about the properties of CourseDetails being treated differently - you're updating some but filtering by others, which isn't what I'd expect from the name of the method.
I would recommend putting only the ones you're updating in `IHasCourseOptions` and having any others as parameters. It may require a name change to make that work right too.

Reply:
It would be nice if the name made it clear what a true / false return value meant too.